recipes/r-dimsum/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = '1.1.3' %}
+{% set github = "https://github.com/lehner-lab/DiMSum" %}
+
+package:
+  name: r-dimsum
+  version: "{{ version }}"
+
+source:
+  url: "{{ github }}/archive/v{{ version }}.tar.gz"
+  md5: e28ac1a49bafb7fffa0ce197e7548054 
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+  host:
+    - r-base =3.6
```

Comment:
Is there a reason this needs R 3.6 rather than 4.0?

Reply:
I haven't tested with R 4.0 yet, but my plan is to do this for the next update.

Reply:
Is this ok? Or is it mandatory that this package is available for R 4.0?

Reply:
I have now changed this to >=3.6